014_squaresum.js

```diff
@@ -3,7 +3,7 @@ let number = readline.question("enter the Number :");
 const sumSquare = (num) => {

```

Comment:
Logic is wrong. If I input 2450 as input, output gives false. But 2450 can be expressed as squares of two whole number.